tfhe/src/high_level_api/integers/signed/inner.rs

```diff
@@ -207,7 +208,7 @@ impl RadixCiphertext {
     }
 
     #[inline]
-    #[allow(clippy::unused_self)]
+    #[allow(clippy::unused_self, clippy::needless_pass_by_ref_mut)]
     pub(crate) fn move_to_device_of_server_key_if_set(&mut self) {
         #[cfg(feature = "gpu")]
         if let Some(device) = global_state::device_of_internal_keys() {
```

Comment:
does this require the cfg(feature) actually ?

Reply:
Features defined in Cargo.toml are know to cargo so are marked as expected

Reply:
what I mean is, does the move to device require the GPU feature here ?

Reply:
Removed